.yaspellerrc.json

```
@@ -40,6 +40,22 @@
     "unstaking",
     "AvroBinary",
     "ipfs",
-    "IPFS"
+    "IPFS",
+    "mainnet",
+    "notated",
+    "sourcing",
+    "coinless",
+    "democratizatin",
+    "uFRQCY",
+    "Milli",
+    "mFRQCY",
+    "schemas",
+    "Crowdloan",
+    "roadmap",
+    "incentivization",
+    "scalability",
+    "Polkadot’s",
+    "for dApps",
```

I would think this should work instead, but not sure we want to use the term dApps.
```suggestion
    "dApps",
```